test/systemtest/earlgrey/test_sim_verilator.py

```diff
@@ -427,7 +427,7 @@ def test_openocd_basic_connectivity(tmp_path, bin_dir, topsrcdir, openocd):
     with the core or the system (bus) is performed.
     """
     # Run a simulation (bootrom only, no app beyond that)
-    sim_path = bin_dir / "hw/top_earlgrey/Vchip_earlgrey_verilator"
+    sim_path = bin_dir / "hw/top_earlgrey/Vchip_earlgrey_verilator_tb"
```

Comment:
Could you run Verilator sims via dvsim as well to check nothing is broken?
```
$ ./util/dvsim/dvsim.py hw/top_earlgrey/dv/verilator_sim_cfg.hjson -i dif_aes_smoketest --local
```

Reply:
Ohh, we should also move `hw/top_earlgrey/dv/verilator_sim_cfg.hjson` to `hw/top_earlgrey/dv/verilator` while we are at it.

Reply:
o yeah i can try that. I did some tests through non-dvsim and it was okay.
Let me see if CI flags anything else first, then i'll go and do the DV hierarchy changes.